lib/templates/route-by-port.ejs

```diff
@@ -0,0 +1,54 @@
+error_log error.log;
+pid nginx.pid;
+
+events {
+  worker_connections  4096;  ## Default: 1024
+}
+
+http {
+  access_log access.log combined;
+  proxy_redirect          off;
+  proxy_set_header        Host            $host;
+  proxy_set_header        X-Real-IP       $remote_addr;
+  proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+  proxy_connect_timeout   90;
+  proxy_send_timeout      90;
+  proxy_read_timeout      90;
+  proxy_buffers           32 4k;
+
+<%
+  for(var s in services) {
+    if (!services.hasOwnProperty(s)) continue;
+    var svc = services[s];
+    if (svc.endpoints.length > 0) {
+%>
```

Comment:
Already went through and places the -%> where it looked ok. config should be user readable as is.